src/desktop/src/ui/views/settings/Node.js

```diff
@@ -118,12 +118,15 @@ const NodeSettings = ({ customNodes, generateAlert, loading, nodes, settings, ac
         if (quorumEnabled !== settings.quorumEnabled || quorumSize !== settings.quorumSize) {
             actions.updateQuorumConfig({ enabled: quorumEnabled, size: quorumSize });
         }
+        if (powNode !== settings.powNode) {
+            actions.setPowNode(powNode);
+        }
+        if (powNode.length ^ this.props.powNode.length) {
+            this.props.changePowSettings();
```

Comment:
```suggestion
            actions.changePowSettings();
```

>125:13  error  Stateless functional components should not use this  react/no-this-in-sfc

https://buildkite.com/iota-foundation/trinity-desktop-prs/builds/4935#8a8cb4cb-3e4f-495a-93f3-f5a1f8ff734f/54-260